### PR TITLE
Guide on representing formulary using MedicationKnowledge

### PR DESCRIPTION
**Pull request by rahul1**

### CODE REVIEW

File: packages/docs/docs/medications/formulary.md [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+            "code": "imprintcd"
+          }
+        ]
+      },
+      "valueString": "Tylenol"
+    }
+  ],
+  "amount": {
+    "value": 100,
+    "unit": "{tbl}"
```
Comment (codyebberson) [inline packages/docs/docs/medications/formulary.md:94]: Is this a standard unit?

We should add `system` property here to clarify

File: packages/docs/docs/medications/formulary.md [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+
+For each ingredient, the `strength` is stored as a ratio of the total volume (e.g. 1mg / 100mg). The field `ingredient.active` indicates which of the listed ingredients are active.
+
+```ts
+{
+"resourceType": "MedicationKnowledge",
+// Dispense 200g total
+"amount": {
+  "value": 200,
+  "unit": "g"
```
Comment (codyebberson) [inline packages/docs/docs/medications/formulary.md:153]: Should this be `mg`?

File: packages/docs/docs/medications/formulary.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+        {
+          "system": "http://www.nlm.nih.gov/research/umls/rxnorm",
+          "code": "1992"
+        }
+      ]
+    },
+    "strength": {
+      "numerator": {
+        "value": 0.075,
+        "unit": "g"
```
Comment (codyebberson) [inline packages/docs/docs/medications/formulary.md:215]: Units and values do not match text.

File: packages/docs/docs/medications/formulary.md [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+
+For a deeper understanding of RxNorm, refer to our [RxNorm guide](./medication-codes).
+
+## Drug Characteristics
+
+Details about the physical characteristics of a drug, such as its color, shape, size, and imprint, can be found in the `drugCharacteristics` section of the [`MedicationKnowledge`](/docs/api/fhir/resources/medicationknowledge) resource. The [Medication knowledge characteristic code](http://hl7.org/fhir/R4/valueset-medicationknowledge-characteristic.html) valueset can be a handy reference for examples of `MedicationKnowledge.drugCharacteristics.type`.
+
+Other metadata fields to note include `packaging`, `doseForm`, and `intendedRoute`.
+
+- The `packaging` attribute represents the type of packaging and quantity of the drug in the package. For instance, `'{tbl}'` is used to indicate a tablet. More details can be found in the [MedicationKnowledge Package Type Codes](http://hl7.org/fhir/R4/valueset-medicationknowledge-package-type.html) and the guide to [units of measure](https://terminology.hl7.org/4.0.0/ValueSet-v3-UnitsOfMeasureCaseSensitive.html).
```
Comment (codyebberson) [inline packages/docs/docs/medications/formulary.md:33]: `tbl` does not appear in this doc http://hl7.org/fhir/R4/valueset-medicationknowledge-package-type.html

**Review by codyebberson — COMMENTED** [review]
Text content looks great.

A bunch of the example dosages and units seem off

File: packages/docs/docs/medications/formulary.md (hunk 4 above)
Comment (rahul1) [reply packages/docs/docs/medications/formulary.md:33]: Good catch - this was a chatgpt-ism I thought I had edited out

File: packages/docs/docs/medications/formulary.md (hunk 1 above)
Comment (rahul1) [reply packages/docs/docs/medications/formulary.md:94]: It's a UCUM unit, which is "standard" within FHIR. I'll add the system string

File: packages/docs/docs/medications/formulary.md (hunk 2 above)
Comment (rahul1) [reply packages/docs/docs/medications/formulary.md:153]: Will update the text above to indicate `g`. 

Grams is actually the right amount for this example

File: packages/docs/docs/medications/formulary.md [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+  - medication
+---
+
+# Modeling a Formulary
+
+## Introduction
+
+A formulary is a catalog of drugs offered by your organization. When implementing a custom EMR, you may want to include essential metadata about these drugs to help prescribing physicians, pharmacists, and patients.
+
+The principal resource for building a digital formulary is the [`MedicationKnowledge`](/docs/api/fhir/resources/medicationknowledge) resource. This guide aims to explore the most pertinent attributes of [`MedicationKnowledge`](/docs/api/fhir/resources/medicationknowledge) for digital health providers.
```
Comment (reshmakh) [inline packages/docs/docs/medications/formulary.md:14]: Consider one more sentence:  In a formulary implementation MedicationKnowledge represents a type of type of medication that can be ordered.

File: packages/docs/docs/medications/formulary.md [hunk 6]
```diff
@@ -0,0 +1,298 @@
...
+
+</details>
+
+## Drug Classifications and Regulations
+
+The `MedicationKnowledge.productType` field can be used to categorize the drug within the formulary, potentially along multiple dimensions. This field can also be used to describe:
+
+- Whether a drug is generic or branded
+- Whether the drug is prescribable
+- Whether the drug is dispensable
```
Comment (reshmakh) [inline packages/docs/docs/medications/formulary.md:132]: May want to provide a little context on what dispensable means in this context and an example.

File: packages/docs/docs/medications/formulary.md [hunk 7]
```diff
@@ -0,0 +1,382 @@
...
+  - pharmacy
+tags:
+  - administration
+  - medication
+  - pharmacy
+---
+
+# Modeling a Formulary
+
+A "formulary" refers to a catalog of drugs offered by your organization. When implementing a custom EMR, Digital Health Clinical administrators often curate a formulary of relevant drugs, along with relevant metadata, to assist prescribing physicians, pharmacists, and patients.
```
Comment (reshmakh) [inline packages/docs/docs/medications/formulary.md:15]: Like this

File: packages/docs/docs/medications/formulary.md [hunk 8]
```diff
@@ -0,0 +1,382 @@
...
+
+This guide covers the basics of how to use the [`MedicationKnowledge`](/docs/api/fhir/resources/medicationknowledge) resource, which represents a type of medication that can be ordered. It will cover:
+
+- How to annotate medications with the proper codes.
+- How to represent physical characteristics about the medication and its package.
+- How to categorize medications in the formulary.
+- How represent compounded medications.
```
Comment (reshmakh) [inline packages/docs/docs/medications/formulary.md:22]: How "to" represent compounded medications

**Review by reshmakh — APPROVED** [review]

File: packages/docs/docs/medications/formulary.md [hunk 9]
```diff
@@ -0,0 +1,382 @@
...
+        },
+      ],
+    },
+  },
+//...
+}
+```
+
+</details>
+
```
Comment (reshmakh) [inline packages/docs/docs/medications/formulary.md:363]: Consider adding a link to a bundle of this resource and prompt people download and upload to their account.

**Review by reshmakh — APPROVED** [review]

**Pull request merged**